src/Context/DrupalErrorCheckApiContext.php

```diff
@@ -14,6 +14,17 @@
  */
 class DrupalErrorCheckApiContext extends RawDrupalContext {
 
+  /**
+   * If watchdog should fail on notice.
+   *
+   * @var boolean $fail_on_notice
+   */
+  protected static $fail_on_notice;
```

Comment:
should be non-static class variable since its only written in a non-static way

Reply:
It is used in a static function checkForWatchdogErrors(), where it can not be used as a class variable